.gitignore

```diff
@@ -3,7 +3,7 @@ syntax: glob
 ### VisualStudio ###
 
 # Tool Runtime Dir
-/[Tt]ools/
+[Tt]ools/
```

Comment:
Can you double check that this change will match only the root /Tools directory and not any of the underlying tools/ directories? 

For example, we have `~/git/wcf/src/System.Private.ServiceModel/tools` - I think this change will ignore that directory.